packages/fxa-auth-server/docs/swagger/auth-server-api.ts

```diff
@@ -123,15 +123,42 @@ export const AUTH_SERVER_API_DESCRIPTION = {
-    | 404         | 176   | Unknown subscription                                                          |
-    | 400         | 177   | Unknown subscription plan                                                     |
-    | 400         | 178   | Subscription payment token rejected                                           |
+    | 400         | 169   | Public clients require PKCE OAuth parameters                                  |
+    | 400         | 170   | Required Authentication Context Reference values could not be satisfied       |
+    | 400         | 171   | Incorrect client_secret                                                       |
+    | 400         | 172   | Unknown authorization code                                                    |
+    | 400         | 173   | Mismatched authorization code                                                 |
+    | 400         | 174   | Expired authorization code                                                    |
+    | 400         | 175   | Public clients require PKCE OAuth parameters                                  |
+    | 404         | 176   | Unknown customer                                                              |
+    | 400         | 177   | Unknown subscription                                                          |
+    | 400         | 178   | Unknown subscription plan                                                     |
+    | 400         | 179   | Subscription payment token rejected                                           |
+    | 400         | 180   | Subscription has already been cancelled                                       |
+    | 400         | 181   | Customer update rejected                                                      |
+    | 400         | 182   | Unknown refresh token                                                         |
+    | 400         | 183   | Invalid or expired confirmation code                                          |
+    | 400         | 184   | Subscription has already been cancelled                                       |
+    | 400         | 185   | Subscription plan is not a valid update                                       |
+    | 400         | 186   | Payment method failed                                                         |
+    | 400         | 187   | User already subscribed                                                       |
+    | 400         | 188   | Failed to find a subscription associated with Stripe source                   |
+    | 400         | 192   | Billing agreement already on file for this customer                           |
+    | 400         | 193   | PayPal payment token is missing                                               |
+    | 400         | 194   | PayPal billing agreement is missing for the existing subscriber               |
+    | 400         | 195   | Account for this email has an active subscription                             |
```

Comment:
The actual error code says "UNVERIFIED_PRIMARY_EMAIL_HAS_ACTIVE_SUBSCRIPTION" which doesn't sound quite the same, but I assume it's just a variable that has evolved over time

Reply:
A few of them seemed like they were updated along the way. "Account for this email has an active subscription" is the correlating error message for errno "UNVERIFIED_PRIMARY_EMAIL_HAS_ACTIVE_SUBSCRIPTION":
https://github.com/mozilla/fxa/blob/1d6006f422f0112ef850caa4295a71aed104690e/packages/fxa-auth-server/lib/error.js#L887-L895